### PR TITLE
add Remote Management category with TigerVNC,  Remmina, and X2go

### PR DESCRIPTION
**Pull request by nix1947**
I have added a remote management category,

### CODE REVIEW

**Comment by n1trux** [comment]
Typo "Remminna" – pls fix. Also I think TigerVNC is way better than RealVNC, because FLOSS (this is a FLOSS list).

Would you mind changing that? :)

**Comment by nix1947** [comment]
Ok,  I will

**Comment by n1trux** [comment]
Please squash your commits and fix the typo in your commit message.

**Comment by nix1947** [comment]
which typo, I didn't get it, could you please guide me

**Comment by n1trux** [comment]
* Re**mm**i**n**a
* Tige**r**VNC

**Comment by n1trux** [comment]
haha, not in the title, but in the commit itself :)

I'd do it myself at this point, but I think that's a good exercise in exactness and creating a useful PR. Please squash your commits into one while you're at it ;)

**Comment by nix1947** [comment]
I have also changed in the commit as well, and squash the commit, into single one, I don't know why it is not showing in pull, but in the local, the commit shown a single one, and commit message is ok, Oh god, what the heck with me, let me try once again.

File: README.md [hunk 1]
```diff
@@ -485,6 +486,12 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
   * [PostgreSQL-XL](http://www.postgres-xl.org/) - Scalable PostgreSQL-based database cluster.
   * [SQLite](http://sqlite.org/) - Library that implements a self-contained, serverless, zero-configuration, transactional SQL DBS.
 
+## Remote Management
+
+* [Tiger VNC](http://tigervnc.org/) - TigerVNC is a high-performance, platform-neutral implementation of VNC (Virtual Network Computing), a client/server application that allows users to launch and interact with graphical applications on remote machines.
+* [Remminna](http://www.remmina.org/wp/) - A Feature rich remote desktop application for linux  and other unixes.
```
Comment (n1trux) [inline README.md]: typo: Remmi**n**na

Comment (nix1947) [inline README.md]: ok got it:)

**Comment by n1trux** [comment]
cool, only one step needed – squash the commits again and we're good to go ;)

**Pull request merged**